app/assets/stylesheets/ideas.scss

```diff
@@ -1,3 +1,10 @@
-// Place all the styles related to the Ideas controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+#deleted {
+  text-align: center;
```

Comment:
Properties should be ordered background, border, display, margin, margin-left, margin-right, text-align, width